backend/benefit/applications/api/v1/views.py

```diff
@@ -154,7 +154,19 @@ def simplified_application_list(self, request):
         )
 
         serializer = self.serializer_class(qs, many=True, context=context)
-        return Response(serializer.data, status=status.HTTP_200_OK)
+
+        order_key = request.GET.get("order_key", "created_at")
+
+        order_by = request.GET.get("order_by", "asc")
+        is_order_descending = order_by.lower() == "desc" or False
```

Comment:
`.lower()` might fail if `order_by` is not string, and that's probably why there's the `or False` at the end? You could define a serializer for the query parameters to handle validation etc. It would handle HTTP responses etc. automatically. [Example](https://tinystruggles.com/posts/drf_recipes/#use-a-serializer-for-the-query-parameters).

Same thing with the `order_key`, what happens if it's not a real key?